Patches/Moyo light in the abyss/Scenario/Moyo_Scenario.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+  <Operation Class="PatchOperationSequence">
+	<operations>
+	  <li Class="PatchOperationFindMod">
+			
+		<mods><li>Moyo-Light in the Abyss</li></mods>
+			
+		<match Class="PatchOperationSequence">
+		<operations>
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ScenarioDef[defName="Moyo_RefugeeScenario"]/scenario/parts</xpath>
+				<value>
+					<li Class="ScenPart_StartingThing_Defined">
+						<def>StartingThing_Defined</def>
+						<thingDef>Ammo_6x18mmCharged</thingDef>
+						<count>216</count>
+					</li>
+					<li Class="ScenPart_StartingThing_Defined">
+						<def>StartingThing_Defined</def>
+						<thingDef>Ammo_Abyssnail</thingDef>
+						<count>240</count>
+					</li>
+				</value>
+			</li>
+			
+			<!--Add material filter to faction for MIB type events-->
+			
+			<li Class="PatchOperationConditional">
+				   <xpath>/Defs/FactionDef[@Name="Moyo_PlayerBase"]/apparelStuffFilter</xpath>

```

Comment:
`Moyo_PlayerBase` appears to be from the base Moyo mod, rather than from Abyss. In Abyss, there is `RedMoyo_PlayerBase`, but I don't know if that means you want to swap that def in, or if these are operations that should go into the base Moyo mod.

Reply:
Removed this entire scenario file as is since it isn't useed.